crates/paimon/src/spec/manifest_file_meta.rs

```diff
@@ -0,0 +1,184 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+use serde_bytes::{Bytes, Deserialize as SerdeBytesDeserialize, Serialize as SerdeBytesSerialize};
+use std::fmt::{Display, Formatter};
+
+/// Metadata of a manifest file.
+///
+/// Impl Reference: <https://github.com/apache/paimon/blob/db8bcd7fdd9c2705435d2ab1d2341c52d1f67ee5/paimon-core/src/main/java/org/apache/paimon/manifest/ManifestFileMeta.java>
+#[derive(PartialEq, Eq, Debug, Clone, Serialize, Deserialize)]
+pub struct ManifestFileMeta {
```

Comment:
Maybe we need `#[serde(rename_all = "camelCase")]`?

Reply:
Camel case not applicable，[link](https://github.com/apache/paimon/blob/06c0f4ec808b7522a7e12d4bde1d188b361833b9/paimon-core/src/main/java/org/apache/paimon/manifest/ManifestFileMeta.java#L104)